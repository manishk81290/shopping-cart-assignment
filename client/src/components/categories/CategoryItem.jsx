import React from "react";
import { Row } from "react-bootstrap";

const CategoryItem = ({ children }) => {
  return (
    <Row md={12} className="cat-card align-items-center">
      {children}
    </Row>
  );
};

export default CategoryItem;
