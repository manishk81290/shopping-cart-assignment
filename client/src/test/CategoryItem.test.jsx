import React from "react";
import CategoryItem from "./../components/categories/CategoryItem";
import ShallowRenderer from "react-test-renderer/shallow";

it("CategoryItem with mock useContext hook", () => {
  const element = new ShallowRenderer().render(<CategoryItem />);
  expect(element).toBeTruthy();
  expect(element).toMatchSnapshot();
});
