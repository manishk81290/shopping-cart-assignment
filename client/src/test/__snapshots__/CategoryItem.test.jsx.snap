// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryItem with mock useContext hook 1`] = `
<Row
  className="cat-card align-items-center"
  md={12}
  noGutters={false}
/>
`;
